treeMap.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div>
+        <svg
+            id="treeMapViz"
+            :viewBox='[0, 0, 100, 100]'
+            class=".viewBoxStyle" >
+            <g 
+                v-for="(leaf, index) in this.treeMapLeaves"

```

Comment:
What is this `treeMapLeaves`? (you have to find out what its structure is from examples on Observable OR some other code source)